### PR TITLE
i18n(zh-cn): update astro-components.mdx

### PR DESCRIPTION
**Pull request by zhengbangbo**
#### What kind of changes does this PR include?

- Translated content

#### Description

update astro-components.mdx

### CODE REVIEW

**Comment by zhengbangbo** [comment]
@liruifengv pls review

**Review by liruifengv — APPROVED** [review]
LGTM

File: src/content/docs/zh-cn/core-concepts/astro-components.mdx [hunk 1]
```diff
@@ -75,20 +62,24 @@ const data = await fetch ('SOME_SECRET_API_URL/users').then (r => r.json ());
 
 如果你在这里写普通的 HTML，你的组件将在任何 Astro 页面上呈现它被导入和使用的 HTML。
 
-然而，Astro 的组件模板语法也支持 **JavaScript 表达式**、**导入的组件**和<strong>[特殊的 Astro 指令](/zh-cn/reference/directives-reference/)</strong>。在组件脚本中定义的数据和值（在页面构建时）可以在组件模板中使用，以产生动态创建的 HTML。
+但是，[Astro 的组件模板语法](/zh-cn/core-concepts/astro-syntax/) 还支持 **JavaScript 表达式**、Astro [`<style>`](/zh-cn/guides/styling/#styling-in-astro) 和 [`<script>`](/zh-cn/guides/client-side-scripts/#在-astro-中使用-script) 标签、**导入的组件**和[**特殊的 Astro 指令**](/zh-cn/reference/directives-reference/)。组件脚本中定义的数据和值可以在组件模板中使用，以生成动态创建的 HTML。
```
Comment (Yan-Thomas) [inline src/content/docs/zh-cn/core-concepts/astro-components.mdx:65]:
```suggestion
但是，[Astro 的组件模板语法](/zh-cn/core-concepts/astro-syntax/) 还支持 **JavaScript 表达式**、Astro [`<style>`](/zh-cn/guides/styling/#在-astro-进行设计) 和 [`<script>`](/zh-cn/guides/client-side-scripts/#在-astro-中使用-script) 标签、**导入的组件**和[**特殊的 Astro 指令**](/zh-cn/reference/directives-reference/)。组件脚本中定义的数据和值可以在组件模板中使用，以生成动态创建的 HTML。
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps, I went ahead and fixed the broken links, so the `imports.mdx` page will be marked as updated, even though it isn't! If someone can update it later, that'd be great. Here are the missing changes: https://github.com/withastro/docs/commits/main/src/content/docs/en/guides/imports.mdx?since=2023-04-04T13:31:19.000Z

**Pull request merged**